frontend/app/(footer)/home/ImageHero.module.css

```
@@ -0,0 +1,108 @@
+  border: 1px solid #1e1e1e;
+  background: transparent;
+  width: 1600px;
+  height: 900px;
+  margin: 120px auto auto;
+  transform: translateX(2%) scale(1.2) rotateX(47deg) rotateY(31deg) rotate(324deg);
+  transform-origin: top left;
+  backface-visibility: hidden;
+  transform-style: preserve-3d;
+  overflow: visible !important;
+  z-index: 5;
+  border-radius: 20px;
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

border-radius is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)